bob-apps/bin/bobd.rs

```diff
@@ -120,6 +148,115 @@ async fn main() {
     }
 }
 
+fn configure_testmode(sub_matches: &ArgMatches) -> Result<(ClusterConfig, NodeConfig), String> {
+    let mut addresses = Vec::with_capacity(1);
+    let port = match sub_matches.value_of("grpc-port") {
+        Some(v) => v.parse().unwrap(),
+        None => 20000
+    };
+    let mut this_node = None;
+    if let Some(node_list) = sub_matches.value_of("nodes") {
+        let available_ips: Vec<String> = NetworkInterface::show().unwrap().into_iter().filter_map(|itf|
+            match itf.addr.unwrap() {
+                Addr::V4(addr) => {
+                    Some(addr.ip.to_string())
+                },
+                _ => None
+        }).collect();
+
+        for (addr, index) in node_list.split(",").zip(0..usize::MAX) {
+            let split = &addr.split_once(":").unwrap();
```

Comment:
Again: return contextual error to the caller

Reply:
Done: [1e08373](https://github.com/qoollo/bob/pull/588/commits/1e0837312a5aa82e7351366ebf65dff582dd9a9c)